ADR/ADR004-store-form-schemas-in-an-api.md

```diff
@@ -0,0 +1,144 @@
+# ADR004: Store form schemas in an API
+
+Date: 2022-02-18
+
+## Status
+
+Proposed
+
+## Context
+
+- Manager/Designer components don't need to be as scalable as the runner
+- Runner needs to scale to number of users
+  - Sustainability, runner not needed during quiet times
+- "Published forms" need to be accessible to the runner
+
+### Option 1 - Forms API
```

Comment:
Wonder how scaling would impact the data store, sharing a datastore between Manager and Runner (via the API) means that the datastore itself will written to/read from by both at the same time, but the runner itself only has to read. Though I suppose caching could mitigate this somewhat - especially if we're just reading published schemas from the runner we could add caching to the API (cache invalidation being the most fun thing)